fw-child/resources/scss/_template.scss

```diff
@@ -287,3 +287,70 @@
     display: none;
   }
 }
+
+//
+// ASIDE MENU
+//
+
+.hierarchical-menu {
+  .collapsible-item {
+    display: flex;
+    flex-wrap: nowrap;
+    justify-content: space-between;
+    width: 100%;
+  }
+
+  .button-expand {
+    background-color: transparent;
+    border: none;
+    color: $secondary;
+    display: inline-block;
+    padding: .375rem .5rem;
+    transition: background-color .2s;
+
+    &:hover {
+      background-color: $gray-400;
```

Comment:
(Sorry, a comment I forgot to add at last review). The hover effect on the menu (defined here and in the `a` below) is not found anywhere else. It feels like a different design just for this menu.

I suggest to leave the default style for the link colors and the hover effects. When we will do the QA on staging, we can reevaluate if we want to change the style.

Reply:
@xfrenette-crim I removed the custom styles for the links and left it as the default. My only concern on this matter is how to make more noticeable the active link in the menu. I changed the active link to use the hover state of the link, but it's not easy to differentiate from the regular links.

Reply:
@domcode-ew I will add this point in the Basecamp discussion. I also pushed a style update for the hover effect of the "collapsible" button.